src/components/FilterableProductTable.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import ProductTable from './ProductTable'
+import SearchBar from './SearchBar'
+import '../App.css'
+
+export class FilterableProductTable extends Component {
+
+    state = {
+        products: this.props.products.data,
+    }
+
+    handleSearch = (value) => {
```

Comment:
For it to work properly you should do it like this. Other wise if you don't have any argument in your function you won't show any products!

```
handleSearch = (value) => {
    const newProducts = value ? this.state.products.filter((item) =>
    item.name.toUpperCase().includes(value.toUpperCase())
  ) : this.props.products.data

    this.setState({
      products: newProducts,
    });
  };
```